std/shared/src/main/scala/cats/effect/std/Hotswap.scala

```diff
@@ -141,12 +140,13 @@ object Hotswap {
             }
 
           override def get: Resource[F, Option[R]] =
-            shared.evalMap { _ =>
-              state.get.map {
-                case Acquired(r, _) => Some(r)
-                case _ => None
-              }
-            }
+            Resource.makeFull[F, Option[R]] { poll =>
+              poll(semaphore.acquire) *>
```

Comment:
Thanks, this implementation is looking good! Since we no longer have the `shared` method I was thinking we can add a comment here.
```suggestion
              poll(semaphore.acquire) *> // acquire shared lock
```

Reply:
Should I revert the changes made to the description comments as well?